src/internal.c

```diff
@@ -6566,6 +6575,14 @@ static int DoCertificate(WOLFSSL* ssl, byte* input, word32* inOutIdx,
         InitDecodedCert(dCert, myCert.buffer, myCert.length, ssl->heap);
         ret = ParseCertRelative(dCert, CERT_TYPE, !ssl->options.verifyNone,
                                 ssl->ctx->cm);
+        if (ret != 0) {
```

Comment:
I don't believe this branch is correct.  What about overrides?  What about meta data from cert even if can't verify for inspection?  Aren't all ret cases checked after this processing?